test/meson.build

```diff
@@ -24,8 +24,9 @@ if install_tests
                 'testsuite-80.units',
         ]
                 # install_subdir() does not handle symlinks correctly which causes a very ugly warning when
-                # installing, so if rsync is available we use that instead. See
-                # https://github.com/mesonbuild/meson/issues/9233.
+                # installing, so if rsync is available we use that instead. TODO: switch to the new option
+                # 'follow_symlinks' once we require Meson 1.2.2 or greater, see:
```

Comment:
It will be in the next *feature* release of meson, which is 1.3.0.

If there is a 1.2.2 it will only contain backports for regression fixes and suchlike, no new APIs.

Reply:
https://github.com/systemd/systemd/pull/29208